doc/book/box/data_model.rst

```diff
@@ -199,18 +195,22 @@ Lua versus MsgPack
             -   tuple (`cdata`_)
             -   ``[12345, 'A B C']``
 
+..  _boolean: http://www.lua.org/pil/2.2.html
+..  _string: http://www.lua.org/pil/2.4.html
+..  _number: http://www.lua.org/pil/2.3.html
+..  _table: http://www.lua.org/pil/2.5.html
+..  _cdata: http://luajit.org/ext_ffi.html#call
+
 ..  note::
 
     MsgPack values have variable lengths.
     So, for example, the smallest number requires only one byte, but the largest number
     requires nine bytes.
 
-..  _nil: http://www.lua.org/pil/2.1.html
-..  _boolean: http://www.lua.org/pil/2.2.html
-..  _string: http://www.lua.org/pil/2.4.html
-..  _number: http://www.lua.org/pil/2.3.html
-..  _table: http://www.lua.org/pil/2.5.html
-..  _cdata: http://luajit.org/ext_ffi.html#call
+..  note::
+    
+    The Lua `nil <http://www.lua.org/pil/2.1.html>`_ type is encoded as MsgPack ``nil`` but
+    decoded as ``msgpack.NULL``. For this reason, Tarantool uses a special constant, :ref:`box.NULL <box-null>`.
```

Comment:
This sounds to me like `msgpack.NULL` and `box.NULL` are different things while they are actually [synonyms](https://github.com/tarantool/tarantool/blob/7572b5c6c5dd6274c712964ebc52e02f9cae6416/src/box/lua/schema.lua#L3675). I'd write simply

```
The Lua `nil <http://www.lua.org/pil/2.1.html>`_ type is encoded as MsgPack ``nil`` but
decoded as :ref:`msgpack.NULL <box-null>`.
```